<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Розв'язання СЛАР</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelSLAR">
                 <property name="text">
                  <string>СЛАР</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="linearSystemInput">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>15</number>
                 </attribute>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>17</number>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="horizontalSpacing">
                <number>6</number>
               </property>
               <item row="3" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Коефіціент релаксації (0,2)</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLineEdit" name="relaxFactorEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="palette">
                  <palette>
                   <active>
                    <colorrole role="Base">
                     <brush brushstyle="SolidPattern">
                      <color alpha="255">
                       <red>230</red>
                       <green>230</green>
                       <blue>230</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </active>
                   <inactive>
                    <colorrole role="Base">
                     <brush brushstyle="SolidPattern">
                      <color alpha="255">
                       <red>230</red>
                       <green>230</green>
                       <blue>230</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </inactive>
                   <disabled>
                    <colorrole role="Base">
                     <brush brushstyle="SolidPattern">
                      <color alpha="255">
                       <red>240</red>
                       <green>240</green>
                       <blue>240</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </disabled>
                  </palette>
                 </property>
                 <property name="text">
                  <string>1.14</string>
                 </property>
                 <property name="frame">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton_4">
                 <property name="font">
                  <font>
                   <pointsize>7</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Згенерувати з діаг. переваг</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="font">
                  <font>
                   <pointsize>7</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Згенерувати</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>Скинути</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="3">
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="nEdit">
                    <property name="frame">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Кількість змінних</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="text">
                     <string>Змінити</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1">
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_9" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Результат</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QTableWidget" name="resultTable">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                   </layout>
                   <zorder>resultTable</zorder>
                   <zorder>label_4</zorder>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QGridLayout" name="gridLayout">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item row="5" column="0">
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Гауса-Зейделя</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="2">
                     <widget class="QLabel" name="label_14">
                      <property name="text">
                       <string>Ітерації</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="3" colspan="2">
                     <widget class="QLabel" name="label_19">
                      <property name="text">
                       <string>Час</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="prec1">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>Релаксація</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2" rowspan="2" colspan="3">
                     <widget class="QLineEdit" name="precEdit">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="frame">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="3" colspan="2">
                     <widget class="QLabel" name="time1">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="2">
                     <widget class="QLabel" name="iter2">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="2">
                     <widget class="QLabel" name="iter1">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Обмеження</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0" colspan="2">
                     <widget class="QRadioButton" name="iterRadio">
                      <property name="text">
                       <string>Кількість ітерацій</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="2">
                     <widget class="QRadioButton" name="precRadio">
                      <property name="text">
                       <string>Точність</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Якобі</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="1">
                     <widget class="QLabel" name="prec2">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="3" colspan="2">
                     <widget class="QLabel" name="time2">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="1">
                     <widget class="QLabel" name="prec3">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="2">
                     <widget class="QLabel" name="iter3">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>Точність</string>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="1" rowspan="2">
                     <widget class="QLabel" name="prec4">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="3" rowspan="2" colspan="2">
                     <widget class="QLabel" name="time4">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="3" colspan="2">
                     <widget class="QLabel" name="time3">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="0" rowspan="2">
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>Точний метод</string>
                      </property>
                     </widget>
                    </item>
                    <item row="7" column="2" rowspan="2">
                     <widget class="QLabel" name="iter4">
                      <property name="text">
                       <string>1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="9" column="4">
                     <widget class="QPushButton" name="calcButton">
                      <property name="text">
                       <string>Обчислити</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                   <widget class="QLineEdit" name="iterEdit">
                    <property name="geometry">
                     <rect>
                      <x>200</x>
                      <y>50</y>
                      <width>113</width>
                      <height>20</height>
                     </rect>
                    </property>
                    <property name="visible">
                     <bool>false</bool>
                    </property>
                    <property name="frame">
                     <bool>false</bool>
                    </property>
                   </widget>
                   <zorder>label_3</zorder>
                   <zorder>precRadio</zorder>
                   <zorder>iterRadio</zorder>
                   <zorder>precEdit</zorder>
                   <zorder>iterEdit</zorder>
                   <zorder>label_2</zorder>
                   <zorder>label_6</zorder>
                   <zorder>label_7</zorder>
                   <zorder>label_8</zorder>
                   <zorder>label_9</zorder>
                   <zorder>prec1</zorder>
                   <zorder>prec2</zorder>
                   <zorder>prec3</zorder>
                   <zorder>prec4</zorder>
                   <zorder>label_14</zorder>
                   <zorder>iter1</zorder>
                   <zorder>iter4</zorder>
                   <zorder>iter3</zorder>
                   <zorder>iter2</zorder>
                   <zorder>label_19</zorder>
                   <zorder>time3</zorder>
                   <zorder>time1</zorder>
                   <zorder>time2</zorder>
                   <zorder>time4</zorder>
                   <zorder>calcButton</zorder>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="menuSave"/>
    <addaction name="separator"/>
    <addaction name="menuClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Довідка</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openFile">
   <property name="text">
    <string>Відкрити файл</string>
   </property>
  </action>
  <action name="menuSave">
   <property name="text">
    <string>Зберегти результат</string>
   </property>
  </action>
  <action name="menuClose">
   <property name="text">
    <string>Вийти</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
